src/translations/th.json

```diff
@@ -0,0 +1,293 @@
+                "p4": "หากต้องการเพิ่มวิดเจ็ตใหม่ ให้คลิกปุ่มบวก หรือคลิกเครื่องหมายถูกเพื่อออกจากโหมดแก้ไข เมื่อคุณเพิ่มวิดเจ็ตแรกแล้ว การเริ่มต้นใช้งานนี้จะหายไป คุณสามารถเพิ่มวิดเจ็ตได้ในขณะนี้และสำรวจส่วนขยายด้วยตัวคุณเอง หรือหากคุณต้องการทราบข้อมูลเพิ่มเติมอีกเล็กน้อยเกี่ยวกับคุณลักษณะต่างๆ ที่ Anori มีให้ ให้คลิก 'ถัดไป'"
+            },
+            "folders": {
+                "title": "โฟลเดอร์",
+                "p1": "ตามค่าเริ่มต้น วิดเจ็ตทั้งหมดของคุณจะถูกเพิ่มไปยังโฟลเดอร์ 'หน้าแรก' อย่างไรก็ตาม คุณยังสามารถจัดระเบียบวิดเจ็ตของคุณในโฟลเดอร์ต่างๆ ตัวอย่างเช่น คุณสามารถมีโฟลเดอร์ที่แตกต่างกันสำหรับวิดเจ็ตงานและส่วนบุคคล หรือคุณสามารถมีโฟลเดอร์แยกต่างหากสำหรับแต่ละโครงการ",
+                "p2": "ในการจัดการโฟลเดอร์ ให้คลิกที่ไอคอนรูปเฟืองที่มุมล่างซ้าย การดำเนินการนี้จะเปิดการตั้งค่าซึ่งคุณสามารถปรับแต่ง Anori ตามที่คุณต้องการได้ คลิกที่ 'โฟลเดอร์' ที่นั่นเพื่อเปิดตัวจัดการโฟลเดอร์"
+            },
+            "shortcuts": {
+                "title": "ทางลัด",
+                "p1": "การดำเนินการหลายอย่างใน Anori ทำได้โดยใช้แป้นพิมพ์ลัด",
+                "p2": "วิดเจ็ตมากมายให้คุณโต้ตอบกับพวกเขาโดยใช้เมนูคำสั่ง หากต้องการเปิดเมนู ให้กด {{metaKey}}+K เมื่อคุณเพิ่มวิดเจ็ตสองสามรายการแรก คุณจะสามารถใช้พลังทั้งหมดเพื่อเปิดบุ๊กมาร์ก ค้นหา หรือทำเครื่องหมายงานว่าเสร็จสิ้นโดยใช้เพียงแป้นพิมพ์ของคุณ",
+                "p3": "ตามค่าเริ่มต้น เมื่อคุณเปิดแท็บใหม่ แถบค้นหาของเบราว์เซอร์จะทำงาน Anori ไม่ได้รับเหตุการณ์ใดๆ เมื่อคุณพิมพ์ที่นั่น ดังนั้นหากต้องการใช้ทางลัด คุณจะต้องย้ายโฟกัสไปที่หน้า Anori ก่อน แล้วจึงกดปุ่มทางลัด หากต้องการย้ายโฟกัส คุณสามารถกด Tab หรือคลิกที่ใดก็ได้บนหน้า",
+                "p4": "หากคุณต้องการดูรายการทางลัดทั้งหมด คุณสามารถหาสูตรโกงได้ในส่วน 'ความช่วยเหลือ' ของการตั้งค่า"
+            },
+            "customization": {
+                "title": "การปรับแต่ง",
+                "p1": "แม้ว่า Anori จะให้คุณสร้างหน้าเริ่มต้นของคุณเองจากวิดเจ็ต แต่ก็สามารถปรับแต่งได้มากขึ้น",
+                "p2": "คุณสามารถเปลี่ยนชื่อแท็บใหม่ เปลี่ยนชุดสีพร้อมกับภาพพื้นหลัง และอัปโหลดไอคอนที่คุณกำหนดเองเพื่อใช้ในวิดเจ็ต (นอกเหนือจากไอคอนในตัวกว่า 10,000 ไอคอน)",
+                "p3": "และนี่ยังไม่ถึงครึ่งของตัวเลือกด้วยซ้ำ คุณอาจต้องการตรวจสอบการตั้งค่าเพื่อดูทั้งหมด"
+            },
+            "analytics": {
+                "title": "การวิเคราะห์",
+                "p1": "เราเกือบจะเสร็จแล้ว แต่มีสิ่งสุดท้ายอย่างหนึ่ง โดยค่าเริ่มต้น Anori จะไม่รวบรวมข้อมูลการติดตามใดๆ และไม่ส่งสิ่งใดไปยังเซิร์ฟเวอร์ระยะไกล แต่ฉันใช้การวิเคราะห์ที่ส่งโดยผู้ใช้โดยสมัครใจเพื่อดูว่าฟีเจอร์ใดถูกใช้มากที่สุดและสิ่งใดที่ต้องมีการขัดเกลาเล็กน้อย",
+                "p2": "ดังนั้น ฉันขอให้คุณเปิดใช้งานการส่งการวิเคราะห์ Anori ไม่ส่งข้อมูลส่วนตัวใดๆ (เช่น บันทึกย่อหรือ URL ที่คุณบันทึกไว้) ส่งเพียงตัวชี้วัดว่าคุณใช้ส่วนขยายอย่างไร (เช่น จำนวนโฟลเดอร์ที่คุณมีและความถี่ที่คุณใช้ทางลัด) แต่สิ่งนี้ขึ้นอยู่กับคุณ Anori จะทำงานตามปกติแม้ว่าคุณจะไม่ได้เปิดใช้งานการวิเคราะห์ก็ตาม"
```

Comment:
I just modified `analyticsHintP2` and added `<0>Learn more.</0>` at the end (this is how whole string looks like now https://github.com/OlegWock/anori/blob/master/src/translations/en.json#L48). While you're at it, could you update your translation to include it too?